### ISSUE TITLE
Animated .gif images are shown as static if "Remove animations" accessibility feature is turned on

### ISSUE THREAD

**Issue by alpetukhov** [deckerst/aves#2015]
**Describe the bug**
Animated `.gif` images are shown as static if "Remove animations" accessibility feature is turned on

**To Reproduce**
1. Go to Android's `Settings` -> `Accessibility` -> `Colour and motion`
2. Toggle `Remove animations` to `On`
3. Open up an animated `.gif` image in `Aves`
4. The image is shown as static

**Expected behavior**
The image should be animated

**System information and logs:**
```
Package: deckers.thibault.aves
Installer: dev.imranr.obtainium
Aves version: 1.14.1-izzy, build 16502
Flutter: main 3.41.2
Android version: 13, API 33
Android build: T2SVS33.68-21-8-20
Device: motorola moto g72
Display: pixel ratio=2.5, logical=432.0x960.0, physical=1080.0x2400.0
Support: dynamic colors=true, geocoder=true, HDR=false, wide gamut=true
Mobile services: ready
Connectivity: wifi, vpn
System locales: en_GB, ru_RU
Storage volumes: /storage/emulated/0/
Storage grants: /storage/emulated/0/Download/Telegram/, /storage/emulated/0/DCIM/
Error reporting: false
```

**Additional context**
I don't see this mentioned in https://github.com/deckerst/aves/issues/1102 so I don't think it's intended.

P.S.
Thanks for the amazing and free app! I very appreciate the work you are doing and totally understand that I'm not entitled to the fixes, just reporting an issue I've encountered.

**Comment by deckerst** [comment]
Thanks for the report.

Do you also have the issue when you enable animations at the Android settings level, but disable them in Aves settings?

~~Could you please provide a sample gif with which you have the issue?~~

**Comment by deckerst** [comment]
Interestingly I could reproduce the issue when the animations are disabled from Android, but it's working as expected when animations are disabled from Aves.

**Comment by lalalasupa0** [comment]
> Interestingly I could reproduce the issue when the animations are disabled from Android, but it's working as expected when animations are disabled from Aves.

Happening to me as well. It only plays animated images (webp, gif, png) when "Transition animation scale" in dev options (from device settings) is on at any value but "off" (even if the other two are off), regardless of the Aves's "Remove animation" setting. Using `https://github.com/deckerst/aves/releases/download/v1.14.1/app-play-release.apk`

**Comment by deckerst** [comment]
Related:
- https://github.com/flutter/flutter/pull/176492
